airflow/utils/db.py

```diff
@@ -298,6 +299,16 @@ def resetdb():
     '''
     Clear out the database
     '''
+    if 'YES_I_REALLY_WANT_TO_BURN_DOWN_THE_METASTORE' not in os.environ:
+        print(
+            "\n\nHOLD IT!! Wiping out the (shared) Airflow metastore is almost "
+            "certainly a really terrible idea.\n\n"
+            "If you're trying to debug a backfill, see the Troubleshooting "
+            "section of our backfill docs [1] or ask for help in "
+            "#etl-productivity\n\n"
+            "[1] https://docs.lyft.net/eng/dataplatformdocs/airflow.html#id12"
+        )
+        sys.exit(1)
```

Comment:
This change is fine to prevent people from running `resetdb`. As a follow up, do we want to send an email to the tools alias in situation where our oncalls are running `resetdb`